### PR TITLE
chore(zh-cn): unify 'see also' translation in glossary part & minor fixes and cleanups

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

'see also' translation unified and do some cleanups for links

### Motivation

enforce translation guide

### CODE REVIEW

File: files/zh-cn/glossary/fork/index.md [hunk 1]
```diff
@@ -9,19 +9,12 @@ slug: Glossary/Fork
 
 Fork 在自由和开源软件的开发中很常见。因为使用 Git 或 Github 平台的贡献模式，Fork 是现在比较流行的术语。
 
-## 了解更多
+## 参见
 
-### 各种 "知名的" forks
+- [Fork](<https://zh.wikipedia.org/wiki/Fork_(software_development)>) on Wikipedia
+- [How to fork a GitHub repo](https://help.github.com/articles/fork-a-repo/) (fork as in a Git context)
+- 各种“知名的” fork
```
Comment (jasonren0403) [inline files/zh-cn/glossary/fork/index.md:16]:
```suggestion
- 各种“知名的”fork
```

File: files/zh-cn/glossary/attribute/index.md [hunk 2]
```diff
@@ -7,9 +7,7 @@ slug: Glossary/Attribute
...
 
-## 深入了解
+## 参见
 
-### 技术参考
-
-- [HTML attribute reference](/zh-CN/docs/Web/HTML/Attributes)
-- 关于 HTML's [global attributes](/zh-CN/docs/Web/HTML/Global_attributes) 的信息
+- [HTML 属性参考](/zh-CN/docs/Web/HTML/Attributes)
+- 关于 HTML 的 [全局属性](/zh-CN/docs/Web/HTML/Global_attributes) 的信息
```
Comment (yin1999) [inline files/zh-cn/glossary/attribute/index.md:13]:
```suggestion
- 关于 HTML 的[全局属性](/zh-CN/docs/Web/HTML/Global_attributes)的信息
```

File: files/zh-cn/glossary/blink/index.md [hunk 3]
```diff
@@ -7,9 +7,7 @@ slug: Glossary/Blink
 
 Blink 是一个开源的浏览器布局引擎，由谷歌作为 Chromium 项目的一部分开发（因而也是 Chrome 的一部分）。具体而言，Blink 是一个派生自 {{glossary("WebKit")}} 项目中 WebCore 库的分支，用于处理布局、渲染以及 {{Glossary("DOM")}}。
 
-## 了解更多
-
-### 常识
+## 参见
 
 - Blink 项目 [主页](https://www.chromium.org/blink)（英文）
```
Comment (yin1999) [inline files/zh-cn/glossary/blink/index.md:12]:
```suggestion
- Blink 项目[主页](https://www.chromium.org/blink)（英文）
```

File: files/zh-cn/glossary/developer_tools/index.md [hunk 4]
```diff
@@ -7,18 +7,13 @@ slug: Glossary/Developer_Tools
...
 - [Web development tools](https://zh.wikipedia.org/wiki/Web_development_tools) on Wikipedia
-
-### 技术参考
-
 - [Firefox Developer Tools](https://firefox-source-docs.mozilla.org/devtools-user/index.html) on MDN
 - [Firebug](https://getfirebug.com/) (former developer tool for Firefox)
 - [Chrome DevTools](https://developer.chrome.com/devtools) on chrome.com
-- [Safari Web Inspector](https://developer.apple.com/library/content/documentation/AppleApplications/Conceptual/Safari_Developer_Guide/Introduction/Introduction.html#//apple_ref/doc/uid/TP40007874-CH1-SW1) on apple.com
-- [Edge Dev Tools](https://docs.microsoft.com/en-us/microsoft-edge/f12-devtools-guide) on microsoft.com
+- [Safari Developer Tools](https://support.apple.com/en-gb/guide/safari-developer/dev073038698/mac) on apple.com
```
Comment (yin1999) [inline files/zh-cn/glossary/developer_tools/index.md:18]:
```suggestion
- [Safari 开发者工具](https://support.apple.com/zh-cn/guide/safari-developer/dev073038698/mac)（apple.com）
```

File: files/zh-cn/glossary/tag/index.md [hunk 5]
```diff
@@ -7,13 +7,8 @@ slug: Glossary/Tag
...
 
-### 基本信息
-
-- 维基百科上的[HTML element](https://zh.wikipedia.org/wiki/HTML_element)
-- [HTML Tags on W3](https://www.w3.org/History/19921103-hypertext/hypertext/WWW/MarkUp/Tags.html)
-
-### 技术参考
-
-- [Introduction to HTML](/zh-CN/docs/Learn/HTML/Introduction_to_HTML)
+- 维基百科上的 [HTML element](https://zh.wikipedia.org/wiki/HTML_element)
```
Comment (yin1999) [inline files/zh-cn/glossary/tag/index.md:12]:
```suggestion
- 维基百科上的 [HTML 元素](https://zh.wikipedia.org/wiki/HTML_element)
```

File: files/zh-cn/glossary/tree_shaking/index.md [hunk 6]
```diff
@@ -11,12 +11,7 @@ slug: Glossary/Tree_shaking
...
-## 了解更多
-
-#### 常识
-
-- ["捆绑过程中死代码消除的好处"](https://exploringjs.com/es6/ch_modules.html#_benefit-dead-code-elimination-during-bundling) 在 Axel Rauschmayer 的书中:“探索 JS:模块”
-
-#### 技术参数资料
+## 参见
 
+- ["打包过程中死代码消除的好处"](https://exploringjs.com/es6/ch_modules.html#_benefit-dead-code-elimination-during-bundling) 在 Axel Rauschmayer 的书中:“探索 JS:模块”
```
Comment (yin1999) [inline files/zh-cn/glossary/tree_shaking/index.md:16]:
```suggestion
- [“打包过程中死代码消除的好处”](https://exploringjs.com/es6/ch_modules.html#_benefit-dead-code-elimination-during-bundling)，在 Axel Rauschmayer 的书“探索 JS：模块”中
```

File: files/zh-cn/glossary/url/index.md [hunk 7]
```diff
@@ -11,16 +11,8 @@ slug: Glossary/URL
 
 URL 也可用于文件传输（{{Glossary("FTP")}}） ，电子邮件（{{Glossary("SMTP")}}）和其他应用。
 
-## 了解更多
-
-### 基本知识
+## 参见
 
+- [理解 URL 及其结构](/zh-CN/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL)
+- URL 的语法，在 [URL Living Standard](https://url.spec.whatwg.org/) 中定义。
```
Comment (yin1999) [inline files/zh-cn/glossary/url/index.md:17]:
```suggestion
- URL 的语法，在 [URL 动态标准](https://url.spec.whatwg.org/)中定义。
```

**Review by yin1999 — APPROVED** [review]
LGTM. But let's do some updates for those modified lines.

File: files/zh-cn/glossary/constructor/index.md [hunk 8]
```diff
@@ -28,13 +28,8 @@ function Default() {}
 var defaultReference = new Default();
 ```
 
-## 了解更多
-
-### 基本常识
+## 参见
 
 - 维基百科上的 [构造函数](https://zh.wikipedia.org/wiki/Constructor_%28object-oriented_programming%29)
```
Comment (jasonren0403) [inline files/zh-cn/glossary/constructor/index.md:33]:
```suggestion
- 维基百科上的[构造函数](https://zh.wikipedia.org/wiki/构造器)
```

File: files/zh-cn/glossary/cookie/index.md [hunk 9]
```diff
@@ -11,8 +11,6 @@ Cookie 用于个性化用户的体验。可能包含用户在访问网站时的
 
 `Cookie 可以在服务器级别设置和修改，使用 Set-Cookie` [HTTP header](/zh-CN/docs/Web/HTTP/Cookies), 或者在 JavaScript 中用 [`document.cookie`](/zh-CN/docs/Web/API/Document/cookie).
 
-## 了解更多
-
-### General Knowledge
+## 参见
 
 - [HTTP cookie](https://en.wikipedia.org/wiki/HTTP_cookie) on Wikipedia
```
Comment (jasonren0403) [inline files/zh-cn/glossary/cookie/index.md:16]:
```suggestion
- 维基百科上的 [HTTP cookie](https://zh.wikipedia.org/wiki/Cookie)
```

File: files/zh-cn/glossary/crawler/index.md [hunk 10]
```diff
@@ -7,9 +7,9 @@ slug: Glossary/Crawler
 
 网络爬虫是一种能有条理并且系统化地浏览{{glossary("World Wide Web","Web")}}，以便收集网页数据的程序（通常也被称作“机器人”）。搜索引擎（例如谷歌，必应）利用爬虫来创建索引是网络爬虫最典型的使用案例。
 
-## 了解更多
+## 参见
 
-- 维基百科上关于[Web crawler](https://zh.wikipedia.org/wiki/Web_crawler)的内容
-- [MDN 网络文档术语表](/zh-CN/docs/Glossary)
+- 维基百科上关于 [Web crawler](https://zh.wikipedia.org/wiki/Web_crawler) 的内容
```
Comment (jasonren0403) [inline files/zh-cn/glossary/crawler/index.md:12]:
```suggestion
- 维基百科上的 [Web 爬虫](https://zh.wikipedia.org/wiki/網路爬蟲)
```

File: files/zh-cn/glossary/fork/index.md [hunk 11]
```diff
@@ -9,19 +9,12 @@ slug: Glossary/Fork
 
 Fork 在自由和开源软件的开发中很常见。因为使用 Git 或 Github 平台的贡献模式，Fork 是现在比较流行的术语。
 
-## 了解更多
+## 参见
 
-### 各种 "知名的" forks
+- [Fork](<https://zh.wikipedia.org/wiki/Fork_(software_development)>) on Wikipedia
```
Comment (jasonren0403) [inline files/zh-cn/glossary/fork/index.md:14]:
```suggestion
- 维基百科上的[分叉](<https://zh.wikipedia.org/wiki/分叉_(软件开发)>)
```

File: files/zh-cn/glossary/fork/index.md [hunk 12]
```diff
@@ -9,19 +9,12 @@ slug: Glossary/Fork
...
+- [How to fork a GitHub repo](https://help.github.com/articles/fork-a-repo/) (fork as in a Git context)
```
Comment (jasonren0403) [inline files/zh-cn/glossary/fork/index.md:15]:
```suggestion
- 在 Git 上下文中：[如何分叉 GitHub 仓库](https://help.github.com/articles/fork-a-repo/)
```

File: files/zh-cn/glossary/guard/index.md [hunk 13]
```diff
@@ -5,4 +5,4 @@ slug: Glossary/Guard
 
 {{GlossarySidebar}}
 
-Guard 是 {{domxref("Headers")}} 对象的新特性 (在 {{domxref("Fetch_API","Fetch spec")}} 中定义）, 限制了像 {{domxref("Headers.set","set()")}} 和 {{domxref("Headers.append","append()")}} 方法操作 header 的内容。举个例子，`immutable` guard 意味着 headers 不能被改变。阅读 [Fetch basic concepts: guard](/zh-CN/docs/Web/API/Fetch_API/Basic_concepts#Guard)，了解更多信息
+Guard 是 {{domxref("Headers")}} 对象的新特性 (在 {{domxref("Fetch_API","Fetch spec")}} 中定义）, 限制了像 {{domxref("Headers.set","set()")}} 和 {{domxref("Headers.append","append()")}} 方法操作 header 的内容。举个例子，`immutable` guard 意味着 headers 不能被改变。阅读 [Fetch basic concepts: guard](/zh-CN/docs/Web/API/Fetch_API/Basic_concepts#Guard)，参见信息
```
Comment (jasonren0403) [inline files/zh-cn/glossary/guard/index.md:8]:
```suggestion
Guard 是 {{domxref("Headers")}} 对象的新特性 (在 {{domxref("Fetch_API","Fetch spec")}} 中定义）, 限制了像 {{domxref("Headers.set","set()")}} 和 {{domxref("Headers.append","append()")}} 方法操作 header 的内容。举个例子，`immutable` guard 意味着 headers 不能被改变。阅读 [Fetch basic concepts: guard](/zh-CN/docs/Web/API/Fetch_API/Basic_concepts#Guard)，以了解更多信息。
```

File: files/zh-cn/glossary/head/index.md [hunk 14]
```diff
@@ -7,9 +7,7 @@ slug: Glossary/Head
 
 **Head**是{{glossary("HTML")}}文档的一部分，包含了关于文档的{{glossary("metadata")}}，比如作者、描述以及应用于 HTML 的{{glossary("CSS")}} 或者 {{glossary("JavaScript")}} 文件的链接。
 
-## 了解更多
+## 参见
 
-### HTML head
-
-- MDN 里的{{htmlelement("head")}}元素引用
+- MDN 里的{{htmlelement("head")}}元素参考
```
Comment (jasonren0403) [inline files/zh-cn/glossary/head/index.md:12]:
```suggestion
- MDN 里的 {{htmlelement("head")}} 元素参考
```

File: files/zh-cn/glossary/safe/http/index.md [hunk 15]
```diff
@@ -15,29 +15,24 @@ slug: Glossary/Safe/HTTP
...
-### 一般了解
-
-- HTTP 规范中的 [安全](https://tools.ietf.org/html/rfc7231#section-4.2.1) 定义。
-
-### 技术规范
+## 参见
 
-- 安全方法的详细解释：{{HTTPMethod("GET")}}, {{HTTPMethod("HEAD")}}, {{HTTPMethod("OPTIONS")}}
-- 非安全方法的详细解释：{{HTTPMethod("PUT")}}, {{HTTPMethod("DELETE")}}, {{HTTPMethod("POST")}}
+- HTTP 规范中的 [安全](https://httpwg.org/specs/rfc9110.html#safe.methods) 定义。
```
Comment (jasonren0403) [inline files/zh-cn/glossary/safe/http/index.md:36]:
```suggestion
- HTTP 规范中的[安全](https://httpwg.org/specs/rfc9110.html#safe.methods)定义。
```

**Pull request merged**